demos/html5/ckeditor5/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "demo-html5-ckeditor5",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
In other demos you increased the minor version number instead of the patch number.

Reply:
You're right, Xavi.

There's no version control for the demos, and I've upgraded version of some of them with not much criteria.

Would it be a good idea to track versions manually for the demos? Does it make sense?

Thanks!

Reply:
Fixed on the last commit.